src/bitcoin-cli.cpp

```diff
@@ -293,6 +294,12 @@ int CommandLineRPC(int argc, char *argv[])
             argc--;
             argv++;
         }
+        std::string rpcPass;
+        if (gArgs.GetBoolArg("-stdinrpcpass", false)) {
+            if(!std::getline(std::cin,rpcPass))
```

Comment:
Nit, missing spaces after `if` and `,` and missing `{`.

@laanwj I'm writing a test for bitcoin-cli, maybe add a commit there to fix these?